### DISCUSSION TITLE
v的垃圾回收的优点在哪里那些地方

### DISCUSSION THREAD

**Discussion by putao0525** [vlang/v#17734] [Questions and Answers]
为啥v的垃圾回收不会出现stop the world ,对比go的垃圾回收的优势是啥？

**Comment by i582** [comment]
Я понимаю, что тебе удобнее писать на своем родном языке, но переводя текст на английский ты делаешь это один раз, в то время как если ты пишешь на китайском, каждому пользователю приходится переводить текст на свой язык, незачем тратить время людей понапрасну. Это уже не в первый раз, когда тебя просят об этом. Пожалуйста, используй переводчик перед созданием новой дискуссии или issue.

**Reply by putao0525** [reply]
Хорошо, в следующий раз я обращу внимание

**Reply by xiusin** [reply]
I always ask Baidu to translate questions, and finally ask GPT foreigners if they can understand, fearing that the meaning is wrong, haha. (我每次都是让百度翻译问题，最后问一下GPT外国人能不能理解，怕意思错了， 哈哈。)

**Comment by ghost** [comment]
Я думаю, мой коллега считает, что вам стоит писать сообщения на английском языке, потому что это поможет именно вам проследить, чтобы вы остались поняты. Когда вы в переводчике пишите на китайском, переводя на английский, вы сразу же можете подобрать слова немного иначе и лучше выразить мысль, если видите, что при обратном переводе с английского на китайский появились какие-то неточности. Но когда вы пишите только на китайском, гарантия правильного донесения мысли испаряется.

Расширения для перевода текста в браузере сейчас есть почти у каждого, но позволить понять ваши мысли правильно можете только вы.

**Comment by JalonSolov** [comment]
The replies might have been more effective in English. Using a completely different non-English language asking for something to be translated to English seems... odd.  :-)

**Comment by Wajinn** [comment]
Like I'm totally fine (and maybe others too) with people using their native language or non-English, but seems much more polite to also provide an English translation too, for the other users on the forum.  Google translate or DeepL, are among the possible choices to help translate.  Also, in the future, others looking for help might be doing a search on discussion titles.  If the titles and threads are in many different languages, it will be problematic.  At least if there is an English translation, along with the other language, things can be more easily found.  The expectation is the search on discussion threads will be conducted in English.
